src/manifest_fenix.json

```diff
@@ -0,0 +1,70 @@
+{
+  "manifest_version": 2,
+  "name": "__MSG_extensionName__",
+  "default_locale": "en_US",
+  "version": "15",
+  "author": "Firefox",
+  "applications": {
+    "gecko": {
+      "id": "secure-proxy@mozilla.com",
+      "strict_min_version": "68.0.1",
+      "update_url": "https://private-network.firefox.com/files/updates.json"
+    }
+  },
+
+  "homepage_url": "https://private-network.firefox.com",
+
+  "description": "__MSG_extensionDescription__",
+  "browser_action": {
```

Comment:
I don't believe this browser_action can be included just yet with GeckoView. Support is coming for it though.